rs/nns/integration_tests/BUILD.bazel

```diff
@@ -324,46 +307,6 @@ rust_ic_test(
     deps = DEPENDENCIES + DEV_DEPENDENCIES,
 )
 
-# To run this test,
-#
-#     bazel \
-#         test \
-#         --test_env=SSH_AUTH_SOCK \
-#         //rs/nns/integration_tests:copy_inactive_neurons_to_stable_memory_test
-#
-# The only unusual thing in this command is `--test_env=SSH_AUTH_SOCK`. That causes the
-# SSH_AUTH_SOCK environment variable to be "forwarded" from your shell to the sandbox where the test
-# is run. This authorizes the test to download the test data.
-#
-# Additionally, the following flags are recommended (but not required):
-#
-# --test_output=streamed
-# --test_arg=--nocapture
-#
-# These let you watch the progress of the test, rather than only being able to see the output only
-# at the end.
-#
-# See the .bazelrc for more configuration information.
-rust_ic_test(
-    name = "copy_inactive_neurons_to_stable_memory_test",
```

Comment:
I'm fine with deleting _this test_ but I also know we frequently used it to run individual experiments.  Maybe we should remove the guts of the test, and just give it a benign name so its purpose as a piece of example code / experimentation is clear?

Reply:
Yes, I have used upgrade_canister_with_golden_nns_state for one off experiments multiple times. Very convenient, if I do say so myself 😉